draft-xchacha-rfc-draft00.txt

```diff
@@ -15,8 +15,10 @@ Abstract
 
    The eXtended-nonce ChaCha cipher construction (XChaCha) allows for
    ChaCha-based ciphersuites to accept a 192-bit nonce with similar
-   guarantees to the original construction (except a much lower
-   probability of nonce misuse occurring, which is a security boon).
+   guarantees to the original construction, except with a much lower
+   probability of nonce misuse occurring.  This enables XChaCha
+   constructions to be stateless, while retaining the same security
```

Comment:
I thought it may be useful to mention that this is sufficiently large to enable XChaCha to be used statelessly.